lib/libcheri_caprevoke/aarch64/libcaprevoke_md.h

```diff
@@ -0,0 +1,88 @@
+#pragma once
+
+static inline uint64_t
+caprev_shadow_set_fw(uint64_t * __capability fw, void * __capability user_obj,
+		     uint64_t fwm)
```

Comment:
Continuation lines are indented 4 spaces in from their normal indent level, not to line up